dev/com.ibm.ws.webcontainer.servlet.4.0_fat/fat/src/com/ibm/ws/fat/wc/FATSuite.java

```diff
@@ -77,14 +78,14 @@
                 WCApplicationMBeanStatusTest.class,
                 // @Server Annotations
                 WCServletContainerInitializerExceptionTest.class,
-                WCSameSiteCookieAttributeTests.class
+                WCSameSiteCookieAttributeTests.class,
+                WC5JakartaServletTest.class
```

Comment:
We should add this above the @Server annotation comment since this uses the SHARED_SERVER